modules/nested-tabs/assets/js/frontend/handlers/nested-tabs.js

```diff
@@ -236,13 +235,6 @@ export default class NestedTabs extends Base {
 		elementorFrontend.elements.$window.on( 'elementor/nested-tabs/activate', this.reInitSwipers );
 	}
 
-	preventDefaultLinkBehaviourForTabTitle( event ) {
```

Comment:
Have you checked if this function is used in the Mega Menu?

In the Mega Menu with need 'a href' links.

Reply:
Yes, I checked, the function is not not in use.